File: sources/Example_affinity.5.c
/*
* @@name:	affinity.5c
* @@type:	C
* @@compilable:	yes
* @@linkable:	yes
* @@expect:	success
* @@version:	omp_4.0
*/
void work();
int main()
{
#pragma omp parallel proc_bind(master) num_threads(4)
   {
      work();
   }
   return 0;
}

File: sources/Example_affinity.5.f
! @@name:	affinity.5f
! @@type:	F-fixed
! @@compilable:	yes
! @@linkable:	yes
! @@expect:	success
! @@version:	omp_4.0
      PROGRAM EXAMPLE
!$OMP PARALLEL PROC_BIND(MASTER) NUM_THREADS(4)
      CALL WORK()
!$OMP END PARALLEL
      END PROGRAM EXAMPLE
